content/blog/2018_cfp_announcement.md

```diff
@@ -0,0 +1,74 @@
+Title: Call For Proposals is open!
+Date: 2019-04-30 3:28
+Category: 2018
+Slug: announcing-cfp-2018
+Summary: PyCon India 2018 - Call For Proposals is open!
+
+
+The time has come! And we are thrilled to announce the opening of Call for Proposals. We encourage everyone to [submit a proposal](https://in.pycon.org/cfp/pycon-india-2018/proposals/) on any topic they would like to share with the community. 
+
+Here are some of the important dates to keep in mind:
+
+- **1st May**: Proposal submission starts
```

Comment:
This needs to be removed. The start date is not required I guess. Or if required w need to update it when it is announced.

Reply:
Let's remove it then.